sprint-planning/estimation/readme.md

```diff
@@ -1,8 +1,93 @@
-# Estimation
+# Goals for Estimation
 
-## Content TODO's
+When a team adopts agile, as the team works together, the process of estimating
+new work becomes more effective. Over time, they get a sense for how their team
+approaches the stories and the effort that will be required to complete each story.
+With a few successful sprints completed, the team can start to see patterns of
+success and where the team has had failures. The team can start to look at their
+velocity against their estimates and begin to predict with greater accuracy what
+effort will be needed to complete the next story.
 
-Recommend sizing scale such as fibonacci
-Pick stories to establish baseline (example below assumes fibnonacci)
-    - What story represents 5
-    - What story represents 2
+If the makeup of the team is regularly changing, the process of estimation becomes
+challenging as the patterns that drive the estimation process also change. Initially,
+it will take three to five or more sprints to find a good rhythm to the team estimations.
+When the team changes to a new project or a team is reorganized to join with another
+team, the estimation model is essentially reset and a new baseline must be formed.
+
+## Sizing
+

```

Comment:
As we go on to explain the techniques for sizing stories, it is important to add a dependency on the "readiness factor" of the story. Having a clearly defined problem statement and acceptance criteria are critical to an educated estimation exercise. We should make it explicit that a groomed backlog is a prerequisite for estimation.

Reply:
Good point Suroor. What about considering it as part of the planning meeting itself as only ready stories should be considered as candidate to planning? By being on ready state the story is supposed to attend the quality demanded for  content and acceptance criteria. This way, when describing estimate process this condition is already attended in the planning description.

Reply:
I agree that only ready stories should be considered as candidates for planning.

Reply:
I will add a comment that only ready stories should be estimated.